prisma/schema.prisma

```diff
@@ -7,11 +7,62 @@ generator client {
   provider = "prisma-client-js"
 }
 
-model User {
-  username String @id
-  name     String
-  email    String @unique
+model Users {
+  username  String @id
+  firstName String
+  lastName  String
+  email     String @unique
 
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  createdAt    DateTime       @default(now())
+  updatedAt    DateTime       @updatedAt
+  subdomians   Subdomains[]
+  certificates Certificates?
+}
+
+model Subdomains {
+  id          Int             @id @default(autoincrement())
+  username    String
+  name        String
+  type        SubdomainType
+  value       String
+  description String
```

Comment:
I think that `description`, `course`, and `ports` should all be marked optional (i.e., add a `?`).  The rest can be required.